SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetBitReader.cpp

```diff
@@ -15,7 +15,9 @@ FSpatialNetBitReader::FSpatialNetBitReader(USpatialPackageMapClient* InPackageMa
 
 void FSpatialNetBitReader::DeserializeObjectRef(FUnrealObjectRef& ObjectRef)
 {
-	*this << ObjectRef.Entity;
+	int64 EntityId;
```

Comment:
Why was this necessary?

Reply:
`int64` resolves to a different type than `int64_t` (one was `long`, the other `long long`, don't remember which is which though)

Reply:
Oh c++